CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 # Upcoming
 
 ### 🔄 Changed
+- Add `Logger.assertionFailure` to replace `Logger.assertationFailure` which has incorrect name and it's now deprecated [#935](https://github.com/GetStream/stream-chat-swift/pull/935)
```

Comment:
Can we just say "`Logger.assertationFailure` was renamed to `Logger.assertionFailure`"? :) I think it's enough :)

Reply:
done ✅